src/base/bittorrent/torrenthandle.h

```diff
@@ -88,9 +88,9 @@ namespace BitTorrent
     class TrackerEntry;
     struct AddTorrentParams;
 
-    struct AddTorrentData
+    struct CreateTorrentParams
     {
-        bool resumed;
+        bool restored;
```

Comment:
can you add a comment about it?
for example: `bool restored;  // is existing torrent job?`

Reply:
or `bool newTorrent;`? just throwing ideas here.

Reply:
`fastResumed` is incorrect here. It can be *fast* resumed it there is no problem with fastresume data. "Fast resume" means "omit rechecking".